### PR TITLE
CI/CD: disable the enclave-tls openssl testing on alinux2

### PR DESCRIPTION
**Pull request by zhiminghufighting**
Fixes: #1139
Signed-off-by: zhiminghufighting <zhiming.hu@intel.com>

### CODE REVIEW

**Comment by zhiminghufighting** [comment]
Local cicd run and debug log is here:
https://github.com/zhiminghufighting/inclavare-containers-1/actions/runs/959560116

some failure is caused by 0.6.1 container image and sgx1 environment instead of yml file logic.

**Comment by haosanzi** [comment]
Please disable alinux2 test in `pr_enclave.yaml` as well

File: .github/workflows/pr_shelter.yml [hunk 1]
```diff
@@ -17,7 +17,8 @@ jobs:
     strategy:
       matrix:
         sgx: [[self-hosted, SGX1], [self-hosted, SGX2, vsgx]]
-        tag: [ubuntu18.04, centos8.2, alinux2]
+      # tag: [ubuntu18.04, centos8.2, alinux2]
```
Comment (jiazhang0) [inline .github/workflows/pr_shelter.yml:20]: Please add the comment to explain why alinux2 is disabled.

File: .github/workflows/pr_enclave_tls.yml [hunk 2]
```diff
@@ -17,7 +17,12 @@ jobs:
     strategy:
       matrix:
         sgx: [[self-hosted, SGX1], [self-hosted, SGX2, vsgx]]
-        tag: [ubuntu18.04, centos8.2, alinux2]
+      # Openssl 1.1 default install path in alinux2 is different with ubunut and centos and it is nonstandard path, it will cause encalve-tls building error.
```
Comment (jiazhang0) [inline .github/workflows/pr_enclave_tls.yml:20]: plz fix the indent issue.

File: .github/workflows/pr_enclave_tls.yml [hunk 3]
```diff
@@ -17,7 +17,12 @@ jobs:
...
+      # It is not a general soltuion to adapt openssl 1.1 default install path in alinux2 to fix building issue.
```
Comment (jiazhang0) [inline .github/workflows/pr_enclave_tls.yml:21]: s/install/installation/

File: .github/workflows/pr_enclave_tls.yml [hunk 4]
```diff
@@ -17,7 +17,12 @@ jobs:
     strategy:
       matrix:
         sgx: [[self-hosted, SGX1], [self-hosted, SGX2, vsgx]]
-        tag: [ubuntu18.04, centos8.2, alinux2]
+       #Openssl 1.1 default installation path in alinux2 is different with ubunut and centos and it is nonstandard path, it will cause encalve-tls building error.
```
Comment (jiazhang0) [inline .github/workflows/pr_enclave_tls.yml:20]: Use `# ` as the leading of comment line.

Comment (zhiminghufighting) [reply .github/workflows/pr_enclave_tls.yml:20]: Let's follow this file as example in format style:
https://github.com/alibaba/inclavare-containers/blob/master/.github/workflows/manually_inclavare_dev_image.yml

use indent following comment symbol "#" and put comment in section start, it is ok?

Comment (zhiminghufighting) [reply .github/workflows/pr_enclave_tls.yml:20]: https://github.com/alibaba/inclavare-containers/blob/master/.github/workflows/manually_compilation_testing_image.yml
https://github.com/alibaba/inclavare-containers/blob/master/.github/workflows/manually_ubuntu_test_image.yml

Suggest to follow the above format style

**Review by jiazhang0 — APPROVED** [review]

**Pull request merged**